modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -1200,7 +1200,7 @@ public void testGetValuesList() throws Exception {
 		Assert.assertEquals("John", values.get("firstName"));
 		Assert.assertEquals(
 			"listTypeEntryKey3", values.get("listTypeEntryKeyRequired"));
-		Assert.assertEquals(values.toString(), 24, values.size());
+		Assert.assertEquals(values.toString(), 25, values.size());
```

Comment:
Hi @p-albuquerque,

What does this number (25) means?

Let's avoid "magic numbers" by trying to extract this number to a final variable with a name that makes easy the comprehension, talk to me when you desired if you need support to do it.

Thanks!